controllers/rollingupgrade_controller.go

```diff
@@ -193,6 +194,8 @@ func (r *RollingUpgradeReconciler) DrainNode(ruObj *upgrademgrv1alpha1.RollingUp
 	drainTimeout int) error {
 	// Running kubectl drain node.
 	err := r.preDrainHelper(ruObj)
+	spew.Println("err := r.preDrainHelper(ruObj)" , err)
```

Comment:
we can probably also remove this one? I guess a remnant from your debugging 😉

Reply:
This is cleaned up.